### PR TITLE
fix: SfSearchbar and SfInput on safari

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
#963

# Scope of work
fix input search and searchbar on safari (remove native layout)
can be tested on https://zeit.co/adampawlinski/storybook-static/lm6xawdda

# Screenshots of visual changes
# Checklist
 
- [x] No commented blocks of code left
- [ ] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
Input search and searchbar fixed. Number validation added on safari. To do that input type is changed from number to text (only on safari). 
TO DO:  regex doesn't work properly on Safari (comma and dot do not pass)

File: packages/vue/src/components/atoms/SfInput/SfInput.vue [hunk 1]
```diff
@@ -160,7 +161,26 @@ export default {
     type: {
       immediate: true,
       handler: function(value) {
-        this.inputType = value;
+        const ua = navigator.userAgent.toLowerCase();
```
Comment (pspaczek) [inline packages/vue/src/components/atoms/SfInput/SfInput.vue:164]: @AdamPawlinski you can't to that :) it's broke SSR :) because navigator is a client API

Comment (pspaczek) [reply packages/vue/src/components/atoms/SfInput/SfInput.vue:164]: just FYI

Comment (AdamPawlinski) [reply packages/vue/src/components/atoms/SfInput/SfInput.vue]: thanks :)

**Pull request merged**